packages/@adobe/spectrum-css-temp/components/popover/index.css

```diff
@@ -48,7 +48,6 @@ governing permissions and limitations under the License.
 
   outline: none; /* Hide focus outline */
   box-sizing: border-box;
-  overflow: hidden;
```

Comment:
it was the scrollbars...

Reply:
Looks like removing that affects the borders. Here is a screenshot of what it looks like WHCM on Chrome + Edge (respectively):
<img width="231" alt="Screenshot 2023-08-17 at 1 46 21 PM" src="https://github.com/adobe/react-spectrum/assets/44729383/9e14e913-89f2-4643-ad3b-9c2b780ee1eb">
<img width="249" alt="Screenshot 2023-08-17 at 1 48 07 PM" src="https://github.com/adobe/react-spectrum/assets/44729383/4e69b67a-25c7-46db-af66-3a0e3b9ab838">

Whereas this is what it looks like without this proposed change. 
<img width="240" alt="Screenshot 2023-08-17 at 1 48 12 PM" src="https://github.com/adobe/react-spectrum/assets/44729383/013e58cd-8077-4832-92ed-6a3b76da3963">

Reply:
I remember that change being introduced, it was only for the corners as @yihuiliao pointed out from what I remember.

I think that's pretty minor and not sure we can do anything about it without introducing new wrappers which would require more involved testing

Reply:
Can the submenu be portaled outside the popover element into some sibling?

Reply:
Not easily, MenuTrigger renders the Popover, which creates the portal.
Menu is a child of the MenuTrigger, and therefore a child of Popover/PopoverWrapper.
Menu also has a key FocusScope which we render our submenu inside. Bad things happen when we move our portal outside that FocusScope or outside the Popover's FocusScope.

The better alternative, *I think*, would be to wrap the Menu in an extra div with the same border radius as the Popover and apply overflow: hidden on that so it clips the scroll bars. The submenu's portal would be a sibling of this new wrapper instead of the Menu `ul` itself.